src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -36,6 +38,24 @@ public class SecretManager : IDisposable, ISecretManager
         private string _repositoryClassName;
         private DateTime _lastCacheResetTime;
 
+        internal const string AzureFunctionsSignature = "AzFu";
+
+        // Checksum seed for master keys. The string transformation here results in a
+        // a seed that is versioned in the least significant bits (as a ulong):
+        // BitConverter.ToUInt64(Encoding.ASCII.GetBytes("Master00").Reverse().ToArray());
+        // If there is a need to version, use "Master01" as the next string literal.
+        internal const ulong MasterKeySeed = 0x4d61737465723030;
```

Comment:
How about for these three seeds we add SecretManager unit tests that include the code you give and serve as the location where we can keep these comments and sample code. The tests run and are debuggable, modifiable for the future if we need to version. We could have just a simple comment referencing the tests for details?

Reply:
Great suggestion, done.